### ISSUE TITLE
call search panel settings: influxdb mentioned but prometheus is being used

### ISSUE THREAD

**Issue by dpocock** [sipcapture/homer-app#121]
I installed HOMER 7 using Prometheus

When I look at the settings for the default search panel, I see that influxdb is mentioned there.  Prometheus is not an option in the pull down.  The call stats are missing.

**Comment by lmangani** [comment]
@dpocock the defaults can't possible take all options into account and ship with generic examples.
Are you able to add a Prometheus Stats widget and display any data there?

**Comment by dpocock** [comment]
On 14/04/2019 21:02, Lorenzo Mangani wrote:
> @dpocock <https://github.com/dpocock> the defaults can't possible take
> all options into account and ship with generic examples.
> Are you able to add a Prometheus Stats widget and display any data there?

If I try to edit the stats window it only shows InfluxDB in the
pull-down menu.  It doesn't show Prometheus anywhere when I edit the panel.

Now I've found that I can add a new panel for Prometheus, but when I try
to play with the settings for the new panel, I can't seem to find any
metrics.

When I look in the Prometheus web interface (port 9090) there are some
metrics.  I just can't get them in homer-app.

I've attached some screenshots.
